dotcom-rendering/src/components/DesignTag.tsx

```diff
@@ -7,19 +7,18 @@ import {
 	until,
 } from '@guardian/source/foundations';
 import { LinkButton } from '@guardian/source/react-components';
-import { decidePalette } from '../lib/decidePalette';
-import type { Palette } from '../types/palette';
+import { palette } from '../palette';
 
-const tagStyles = (palette: Palette) => css`
-	background-color: ${palette.background.designTag};
-	color: ${palette.text.designTag};
+const tagStyles = css`
+	background-color: ${palette('--design-tag-background')};
+	color: ${palette('--design-tag-text')};
 	display: inline-block;
 	padding: 2px 0 4px 0;
 	${headlineBold20}
 	line-height: 115%;
 	box-shadow:
-		6px 0 0 ${palette.background.headlineTag},
```

Comment:
It's not clear why `palette.background.headlineTag` is used here rather than `palette.background.designTag` as the box shadow needs to match the background colour. It's not used anywhere else so has been removed in this PR.